app/pages/complete-order/routes.js

```diff
@@ -10,8 +11,10 @@ const router = express.Router({ mergeParams: true });
 export const completeOrderRoutes = (authProvider, addContext, sessionManager) => {
   router.get('/', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
     const { orderId } = req.params;
+    sessionManager.saveToSession({ req, key: 'orderId', value: orderId });

```

Comment:
we shouldn't need to save this from session. We'll always have access to this from the url